fastasyncworldedit/configuration/permissions.md

```diff
@@ -0,0 +1,47 @@
+# Permissions
+
+Table of Contents
+
+[🏠 Return to the index](../README.md)
+
+- [Admin permissions](#admin-permissions)
+- [User permissions](#user-permissions)
+- [Region permissions](#region-permissions)
+- [Extended WorldEdit permissions](#extended-worldedit-permissions)
+
+> [TIP] **Info:** For command related permissions, see [here](../basic-commands/main-commands-and-permissions.adoc).
```

Comment:
This isn't working with gitbook's syntax. Take a look at the other pages or the gitbook documentation how to use gitbook's custom markdown flavor. 

Additionally, the page URL is wrong. The adoc files do not exist here. Rather, use a relative URL to the command page.